### DISCUSSION TITLE
Dashboard is broken

### DISCUSSION THREAD

**Discussion by Ryan365Apps** [Labs64/NetLicensing-Community#116] [Ideas & Feature Requests]
https://ui.netlicensing.io/#/dashboard 

does not load

**Comment by r-brown** [comment]
Hello @Ryan365Apps ,
We switched off dashboard component for maintenance and will enable this as soon as revisited version will be available.

**Comment by Ryan365Apps** [comment]
As feedback, it would be nice to have been informed.

**Comment by r-brown** [comment]
Sure, we maintain product [changelog](https://netlicensing.io/wiki/changelog#netlicensing-console-maintenance-and-performance-improvements) and announce all additions or maintenance activities there.

**Comment by Ryan365Apps** [comment]
I'm trying to be helpful to you guys, you have a cool product but if something is broken, a customer shouldn't be expected to check a changelog to see if it was done on purpose. It doesn't feel right. 

Just email out your changelog updates before you do them. 

The same goes for service incidents - be proactive.
